.github/workflows/on-push-master.yml

```diff
@@ -59,13 +82,16 @@ jobs:
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
 
-    - name: Install ruby
```

Comment:
Don't we need that to build the docs below?

Reply:
I moved this up, next to where Java is being set up.

Reply:
Right, missed it... :facepalm: